File: test/js/mario.js
const readlineSync = require('readline-sync');

const height = Number(readlineSync.question('Height: '));

while (height < 1 || height > 8) {
  height = Number(readlineSync.question('Height: '));
}

for (let row = 0; row < height; row++) {
  for (let column = row + 1; column < height; column++) {
    console.log(' '); // print without newline
  }

  for (let hashes = height + row + 1; hashes > height; hashes--) { // right
    console.log('#');
  }

  console.log('  ');

  for (let hashes = height + row + 1; hashes > height; hashes--) { // left
    console.log('#');
  }

  console.log(); // add newline at the end of each row
}

File: Week-1/mario-more/mario.c
//---------------------------------
//From our beloved Land of Papua we write this code
//
//
//mario-more.c
//---------------------------------
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");  //Input height first
    }

    while
    ((height < 1) || (height > 8));  // boolean logic use OR

    // for loop
    for (int row = 0; row < height; row++)

    {

        for (int column = row + 1; column < height; column++)
        {
            printf(" "); //space
        }


        for (int hashes = height + row + 1; hashes > height; hashes--) //right
        {
            printf("#");
        }
        printf("  ");

        for (int hashes = height + row + 1; hashes > height; hashes--) //left
        {
            printf("#");
        }
        printf("\n");
    }
}